middleware/handleAuthentication.js

```diff
@@ -4,6 +4,8 @@ module.exports = function handleAuthentication(req, res, next) {
     next();
   } else {
     res.status(401);
-    res.end();
+    res.json({
+      message: 'Authentication required!',
```

Comment:
Is this part of the body? Does this mean I don't have to do that ghetto response.json() call for only HTTP.OK?